CONTRIBUTING.md

```diff
@@ -1,10 +1,10 @@
 # Quire Contributor Guidelines
 
-**Thank you for your interest in contributing to Quire!**
+**We encourage contributions from our community members. No matter what level of experience you have, we welcome all contributions, big and small.**
 
-Developed by Getty, Quire is a new, multiformat publishing tool available for immediate access and use. Quire is in a limited beta, © J. Paul Getty Trust, and not yet released as open-source software. **For a free license to use Quire, please complete [this form](http://bit.ly/quire-signup).**
+Developed by Getty, Quire is a digital publishing tool that can create dynamic publications in a variety of formats, including web, print, and e-book. Quire is an ideal tool for anyone seeking to publish beautiful, scholarly digital books optimized for visual imagery and designed to ensure their content is widely accessible and stable.
 
-Striving toward becoming a fully open-source project by spring 2022, we encourage contributions from our community members. No matter what level of experience you have, we welcome all contributions, big and small.
+Quire is open source and free to use. View Quire's [3-clause BSD open source license](https://github.com/thegetty/quire/blob/main/LICENSE). Please note: Quire v1 is currently in a pre-release phase. Testing and improvements are ongoing.

```

Comment:
technically, we're in a "release candidate" phase (the `rc` in `1.0.0-rc.0`) rather than a "pre-release" phase, but I don't know if that makes sense?

Reply:
We are following [semantic version](https://semver.org) conventions and using a `pre-release` identifier that is the _release candidate_ id, so each release candidate version is a pre-release version before `v1.0.0`